docs/admin/authorization/rbac.md

```diff
@@ -58,17 +63,20 @@ metadata:
   # "namespace" omitted since ClusterRoles are not namespaced.
   name: secret-reader
 rules:
-  - apiGroups: [""]
-    resources: ["secrets"]
-    verbs: ["get", "watch", "list"]
+- apiGroups: [""]
+  resources: ["secrets"]
+  verbs: ["get", "watch", "list"]
 ```
 
-`RoleBindings` perform the task of granting the permission to a user or set of
-users. They hold a list of subjects which they apply to, and a reference to the
-`Role` being assigned.
+### RoleBindings and ClusterRoleBindings
+
+A role binding grants the permissions defined in a role to a user or set of users.
+It holds a list of subjects (users, groups, or service accounts), and a reference to the role being granted.
+Permissions can be granted within a namespace with a `RoleBinding` object, or cluster-wide with a `ClusterRoleBinding` object.
 
-The following `RoleBinding` assigns the "pod-reader" role to the user "jane"
-within the "default" namespace, and allows jane to read pods.
+A `RoleBinding` may reference a `Role` in the same namespace.
+The following `RoleBinding` grants the "pod-reader" role to the user "jane" within the namespace "default".
```

Comment:
- [ ] Here and throughout, in American English, the punctuation sits within the quotation, even when the quoted text is not a character's speech. Hence, we'd write
> the namespace "default."

Now, with technical documentation where we use quotation marks to indicate significant tokens, placing the punctuation inside the quotation marks can lead to some confusion, such as the following sentence:
> The kind should be "RoleBinding."

Does the kind that one would write in a YAML or JSON object include a period at the end? No, but one could read it that way (too literally).

I got into the habit of placing punctuation outside of quotation marks when typing in plain text documents, but recently I reviewed several style guides and articles on grammar, and found that in American English, the rule is universal and not contextual as it is in British English. Hence, I'm bending my fingers back again.